boards/esp32-ttgo-t-beam/include/periph_conf.h

```diff
@@ -0,0 +1,170 @@
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_esp32_ttgo-t-beam
+ * @brief       Peripheral MCU configuration for TTGO T-Beam board
+ * @{
+ *
+ * TTGO T-Beam is an ESP32 development board with 4 MB Flash that
+ * uses the EPS32 chip directly. It integrates a SemTech SX1276 or SX1278 for
+ * LoRaWAN communication in the 433 MHz or the 868/915 MHz band, respectively.
+ * Additionally, it has an GPS receiver connected via UART on board.
+ *
+ * For detailed information about the configuration of ESP32 boards, see
+ * section \ref esp32_comm_periph "Common Peripherals".
+ *
+ * @note
+ * Most definitions can be overridden by an \ref esp32_app_spec_conf
+ * "application-specific board configuration".
+ *
+ * @file
+ * @author      Yegor Yefremov <yegorslists@googlemail.com>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include <stdint.h>
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+/**
+ * @name    ADC and DAC channel configuration
+ * @{
+ */
+
+/**
+ * @brief   Declaration of GPIOs that can be used as ADC channels
+ *
+ * @note As long as the GPIOs listed in ADC_GPIOS are not initialized as ADC
+ * channels with the \ref adc_init function, they can be used for other
+ * purposes.
+ */
+#ifndef ADC_GPIOS
+#define ADC_GPIOS   { GPIO36, GPIO39, GPIO32, GPIO33, \
+                      GPIO34, GPIO35, GPIO4, GPIO0,\
+                      GPIO2, GPIO13, GPIO25 }
+#endif
+
+/**
+ * @brief   Declaration of GPIOs that can be used as DAC channels
+ *
+ * @note As long as the GPIOs listed in DAC_GPIOS are not initialized as DAC
+ * channels with the \ref dac_init function, they can be used for other
+ * purposes.
+ *
+ * Since all DAC GPIOs are used for board control functions, there are no
+ * GPIOs left that can be used as DAC channels.
+ */
+#ifndef DAC_GPIOS
+#define DAC_GPIOS   { GPIO25 }
+#endif
+/** @} */
+
+
+/**
+ * @name   I2C configuration
+ *
+ * Only I2C interface I2C_DEV(0) is used.
+ *
+ * @note The GPIOs listed in the configuration are only initialized as I2C
+ * signals when module `perpih_i2c` is used. Otherwise they are not
+ * allocated and can be used for other purposes.
+ *
+ * @{
+ */
+#ifndef I2C0_SPEED
+#define I2C0_SPEED  I2C_SPEED_FAST  /**< I2C bus speed of I2C_DEV(0) */
+#endif
+#ifndef I2C0_SCL
+#define I2C0_SCL    GPIO22          /**< SCL signal of I2C_DEV(0) [UEXT1] */
+#endif
+#ifndef I2C0_SDA
+#define I2C0_SDA    GPIO21          /**< SDA signal of I2C_DEV(0) [UEXT1] */
+#endif
+/** @} */
+
+
+/**
+ * @name   PWM channel configuration
+ *
+ * @note As long as the according PWM device is not initialized with
+ * the ```pwm_init```, the GPIOs declared for this device can be used
+ * for other purposes.
+ *
+ * @{
+ */
+/** PWM channels for device PWM_DEV(0) */
+#ifndef PWM0_GPIOS
+#define PWM0_GPIOS  { GPIO25, GPIO0, GPIO2 }
```

Comment:
GPIO14 (LED) should be the first channel in this definition. It would make testing easier.

Reply:
Added GPIO14.